component.css

```diff
@@ -129,6 +141,16 @@ a:visited{
 .text-heading{
     background-color: var(--white);
 }
+.social-links-group{
+    list-style: none;
+    padding: 0rem;
+}
+.social-links{
+    float: left;
```

Comment:
use float is like the old days. We should use flexbox/grid/position instead of float for precise positioning.